kcalcore/icaltimezones.cpp

```diff
@@ -1313,10 +1313,10 @@ QList<QDateTime> ICalTimeZoneSourcePrivate::parsePhase( icalcomponent *c,
   }
 
   // Convert DTSTART to QDateTime, and from local time to UTC
-  const QDateTime localStart = toQDateTime( dtstart );   // local time
   dtstart.second -= prevOffset;
   dtstart.is_utc = 1;
   const QDateTime utcStart = toQDateTime( icaltime_normalize( dtstart ) );   // UTC
+  const QDateTime localStart = utcStart.toLocalTime();  // local time
```

Comment:
For testing purposes it should be enough to rebuild kcalcore on top of it (abi break) and forcing installation temporarily :)

Need to investigate myself these more, but I have a feeling they mostly work around problems, in which case I'd rather have root cause and our packages upgraded. Changelog at least seemed to have a bunch of timezone changes.